Source/Core/DolphinQt/Config/Mapping/IOWindow.h

```diff
@@ -74,7 +76,7 @@ class IOWindow final : public QDialog
   void UpdateOptionList();
   void UpdateDeviceList();
 
-  void UpdateExpression(std::string new_expression);
+  void UpdateExpression(std::string new_expression, bool force);
```

Comment:
Consider taking an enum class (e.g. `UpdateMode` with two values, Normal and Force) because having boolean parameters hurts readability at call sites. When I see a call to UpdateExpression, I have no idea what the boolean does and I have to check the header to know and remember the name. (https://abseil.io/tips/94 explains the reasoning a bit more)

Reply:
Fair point. I assume this is fine now?